packages/server/lib/util/print-run.ts

```diff
@@ -560,3 +560,88 @@ export const printVideoPath = (videoName?: string) => {
     console.log('')
   }
 }
+
+const formatFileSize = (bytes: number) => {
+  if (env.get('CYPRESS_INTERNAL_ENV') === 'test') {
+    return 'XX kB'
+  }
+
+  const KB = 1000
+  const MB = KB * 1000
+  const GB = MB * 1000
+
+  const [size, unit] = (bytes >= (GB) ? [bytes / GB, 'GB'] :
```

Comment:
I wonder if you can leverage [pretty-bytes](https://www.npmjs.com/package/pretty-bytes) here, which we could install in `@packages/server` since we already do it in the cli.